docs/components/badge/android.md

```diff
@@ -0,0 +1,63 @@
+
+## Syntax
+
+```kotlin example
+@Composable
+fun WarpBadge(
+    modifier: Modifier = Modifier,
+    text: String,
+    style: WarpBadgeStyle = WarpBadgeStyle.Neutral,
+    alignmentStyle: WarpBadgeAlignment = WarpBadgeAlignment.None
+)
+```
+
+The default style for the badge is Neutral. 
+
+```kotlin example
+WarpBadge(
+    text = "Success",
+    style = WarpBadgeStyle.Success,
+    alignment = Alignment.BottomStart
+    )
+```
+
+There are a variety of styles and alignements supported for the badge component:
```

Comment:
Just a small thing: remove the extra `e` in `alignements`, so that it spells instead `alignments` :)